README.md

```diff
@@ -84,6 +84,9 @@ Time: 4.45 seconds, Memory: 38.00MB
 OK (2 tests, 7 assertions)
 ```
 
+Notes:
+* If you want to run test with coverage report, use command: `bin/moodle-docker-compose exec webserver phpdbg -qrr vendor/bin/phpunit --coverage-text auth_manual_testcase auth/manual/tests/manual_test.php`
```

Comment:
Hi @kabalin,
how can you get the code coverage report w/o [e.g.](https://phpunit.readthedocs.io/en/7.5/code-coverage-analysis.html) XDebug extension?
Ref.: https://github.com/moodlehq/moodle-php-apache/issues/17

TIA,
Matteo

Reply:
xdebug is not needed. It works from the box if you execute it as shown :) You might be better off to actually use it on mod_forum to see some sensible output, as it got `coverage.php` defined and relevant annotations.

Reply:
TNX for your prompt reply: I never tried since PHPUnit docs tells about the need for an extension.

Will do,
Matteo

Reply:
Thechnically you need one of supported debuggers: Xdebug, PCOV or PHPDBG. PHPDBG is php internal one (and in fact the fastest in code coverage tests, google for details).